packages/gatsby-recipes/src/parser/index.js

```diff
@@ -33,7 +34,11 @@ const applyUuid = tree => {
   return tree
 }
 
-const u = unified().use(remarkParse).use(remarkStringify).use(remarkMdx)
+const u = unified()
+  .use(remarkParse)
+  .use(remarkStringify)
+  .use(remarkMdx)
+  .use(remarkMdxjs)
```

Comment:
Why is there both `remarkMdx` and `remarkMdxjs`?

Reply:
Currently MDX v2 (which this branch is now using) has the js specific parsing split out into a separate package (remark-mdxjs). This will be recombined to all live in `remark-mdx` like it currently does in v1, but we haven't done so quite yet.